src/components/MessageListItem.react.js

```diff
@@ -56,6 +57,8 @@ class MessageListItem extends React.Component {
     let {message} = this.props;
     let stringWithLinks = this.props.message.text;
     let replacedText = parseAndReplace(stringWithLinks);
+    let messageContainerClasses  = cx('message-container '
```

Comment:
@AnshulMalik Yes I couldn't find a way to add multiple classes, please suggest changes :)